src/core/lib/surface/call.cc

```diff
@@ -732,7 +732,7 @@ grpc_error_handle FilterStackCall::Create(grpc_call_create_args* args,
       GPR_ROUND_UP_TO_ALIGNMENT_SIZE(sizeof(FilterStackCall)) +
       channel_stack->call_stack_size;
 
-  Arena* arena = channel->CreateArena();
+  RefCountedPtr<Arena> arena = channel->call_arena_allocator()->MakeArena();
   call = new (arena->Alloc(call_alloc_size)) FilterStackCall(arena, *args);
```

Comment:
Suggest passing arena using `std::move()` here, and then changing line 774 below from `arena.get()` to `call->arena()`.

Reply:
holding this as is to save integration woes